app.py

```diff
@@ -37,13 +41,4 @@ def sign_up():
 	""".format(**data)
 	mail.send(message)
 
-	subject= "Thank you for signing up for Remember Thee"
```

Comment:
@michaelyaacoub, you fixed this merge conflict incorrectly - you removed the work you did to automate sending the welcome email and you removed the route I created. Please undo this - all this should reflect what's on the `main` branch of your `upstream` remote plus any changes you're making for this particular issue.

Reply:
Sorry about that. I hope what I did solved it, let me know if not.